### ISSUE TITLE
Issues on training from scratch

### ISSUE THREAD

**Issue by JimLee1996** [xuexingyu24/License_Plate_Detection_Pytorch#2]
Thanks for your nice work!

 I've noticed that the LPRNet is initialized using a pre-trained weights before training on CCPD. Here is the problem:

When I tried to train my model (another model) from scratch on CCPD, the ctc_loss got down to <=0.01 while accuracy remained <=1%.

I wonder is it necessary to apply pre-training on high quality dataset before CCPD?

**Comment by xuexingyu24** [comment]
Yes  I use pre-trained weight for initialization and that will make training easily reach ~70% accuracy  at first few epochs.   The training might take longer and slower if you just have some random initializations.

**Comment by xuexingyu24** [comment]
Believe I can close this one
